vars/terraformDeployJob.groovy

```diff
@@ -92,6 +92,26 @@ def call(Map config) {
               }
             }
           }
+          stage('Tag Release') {
+            steps {
+              sh "git tag ${versionTag}"
+              sshagent(['github-jenkins']) {
+                sh("git push origin ${versionTag}")
```

Comment:
I'd forgotten the Terraform builds don't allow you to deploy a specific version. Not suggesting you do it as part of this task, but you think we should be adding version tags when the master branch gets built, like we do for ECS and Lambda builds? Would that work for Terraform repos?

Reply:
Not sure. Need to have a think about this.